credentialproviderpackage/pkg/constants/constants.go

```diff
@@ -0,0 +1,41 @@
+package constants
+
+const (
```

Comment:
Why globals? Each thing should have their own constants and most likely all or at least some of these can be lower case for reduced scope